src/main/java/org/folio/service/CirculationSettingsService.java

```diff
@@ -67,4 +71,29 @@ public Future<Response> delete(String circulationSettingsId) {
       .compose(eventPublisher.publishRemoved(circulationSetting))
     );
   }
+
+  private Future<CirculationSetting> updateSettingsValue(CirculationSetting circulationSetting, Throwable throwable) {
```

Comment:
The line must not exceed 100 characters in length

Reply:
done